apps/emqx_authz/etc/acl.conf

```diff
@@ -23,7 +23,7 @@
 %% -type(rule() :: {permission(), who(), access(), topics()} | {permission(), all}).
 %%--------------------------------------------------------------------
 
-{allow, {username, "^dashboard?"}, subscribe, ["$SYS/#"]}.
+{allow, {username, {re, "^dashboard?"}}, subscribe, ["$SYS/#"]}.
```

Comment:
What does this re is expected to match? 🤔  M.b. `"^dashboard$"` is intended?

Reply:
Honestly, I don't know either (M.b it just copied from older versions)